newhelm/main.py

```diff
@@ -33,12 +34,25 @@ def list() -> None:
         display_list_item(f"{test} {test_entry}")
 
 
+@newhelm_cli.command()
+def list_tests() -> None:
+    """List details about all registered tests."""
+    for test, test_entry in TESTS.items():
+        test_obj = test_entry.make_instance()
+        metadata = test_obj.get_metadata()
+        display_header(metadata.name)
+        click.echo(f"command line key: {test}")
+        click.echo(f"Description: {metadata.description}")
+        click.echo()
```

Comment:
Super minor but did we want to capitalize "command"?

Reply:
Done.